src/assets/scripts/client/info/SimClockController.js

```diff
@@ -1,29 +1,51 @@
 import TimeKeeper from '../engine/TimeKeeper';
 import { digits_integer } from '../utilities/radioUtilities';
-import { SELECTORS } from '../constants/selectors';
 import { TIME } from '../constants/globalConstants';
 
 /**
  * Manages a clock that stays in sync with the current game time
- * @class GameClockView
+ * @class SimClockController
  */
-export default class GameClockView {
+export default class SimClockController {
```

Comment:
missing `@class` docblock.  with the questions I asked above about this class, the first place I'd go is the docblock here.  What does this class do and how should I use it, if I need to?